api/types/session.go

```diff
@@ -512,6 +512,8 @@ type NewWebSessionRequest struct {
 	LoginTime time.Time
 	// AccessRequests contains the UUIDs of the access requests currently in use.
 	AccessRequests []string
+	// ClientIP is an IP of the client requesting web session.
+	ClientIP string
```

Comment:
I was expecting to see https://pkg.go.dev/net#IP here
Any reason to use string instead?

Reply:
We store client IP in string format in different places (like `UserCertParams`, `Identity` etc) already, I just kept it consistent

Reply:
Thank you for clarifying :+1: